client-src/elements/chromedash-enterprise-release-notes-page_test.ts

```diff
@@ -280,8 +284,8 @@ describe('chromedash-enterprise-release-notes-page', () => {
 
     // invalid feature requests would trigger the toast to show message
     const toastEl = document.querySelector('chromedash-toast');
-    const toastMsgSpan = toastEl.shadowRoot.querySelector('span#msg');
-    assert.include(toastMsgSpan.innerHTML,
+    const toastMsgSpan = toastEl!.shadowRoot!.querySelector('span#msg');
```

Comment:
Consider using `?` instead of `!` to fix cases where the object could actually be null.